### PR TITLE
Add argo configuration to perform pytorch wandb sweeps

### PR DESCRIPTION
**Pull request by mcgibbon**
This PR adds an argo template to perform Weights and Biases (wandb) hyperparameter sweeps with pytorch models.

Added public API:
- added wandb-sweep-torch-v1 argo template

Coverage reports (updated automatically):
- test_unit: [60%](https://output.circle-artifacts.com/output/job/cd4b7273-c053-4fa8-a814-66b59b342d39/artifacts/0/tmp/coverage/htmlcov-test_unit/index.html)

### CODE REVIEW

File: projects/cyclegan/c48_to_c384/sweep.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+early_terminate:
+  type: hyperband
+  min_iter: 10
+metric:
+  goal: minimize
+  name: val_regularization_loss
+parameters:
+  network.optimizer.kwargs.lr:
+    distribution: log_uniform_values
+    min: 0.000001
```
Comment (AnnaKwa) [inline projects/cyclegan/c48_to_c384/sweep.yaml:24]: FYI I believe when wandb says 'log_uniform',
i) it's natural log, not log10
ii) you have to enter min/max as the exponent, not the full value. e.g. it uses e^min, e^max as the limits.

**Review by AnnaKwa — APPROVED** [review]
LGTM. Watch out for the wandb log uniform entries in the sweep config!

File: projects/cyclegan/c48_to_c384/sweep.yaml (hunk 1 above)
Comment (AnnaKwa) [reply projects/cyclegan/c48_to_c384/sweep.yaml:24]: Nvm, I see there's the 'values'

Comment (mcgibbon) [reply projects/cyclegan/c48_to_c384/sweep.yaml:24]: Yes, I saw the existing example used the non-"_values" version which I found confusing, decided to sidestep the issue.

**Pull request merged**